lib/topological_inventory/orchestrator/health_checker.rb

```diff
@@ -0,0 +1,52 @@
+module TopologicalInventory
+  module Orchestrator
+    class HealthChecker
```

Comment:
I suggest to you `HealthCheck` instead of `HealthChecker` to be consistent with operations repos